packages/v4/src/content/design-guidelines/usage-and-behavior/text-input/text-input.md

```diff
@@ -9,38 +9,39 @@ Use text input to gather free-form text from a user.
+2. **Input field:** Area for user to type in their text input.
+3. **[Help](#help):** Provides contextual assistance like popovers, placeholder text, or helper text, in any area where the user might need additional information to provide the right input.
+4. **[Errors and validation](#errors-and-validation):** Ensures that the correct information is collected before the form can be submitted to the system.
 
 ## Usage
 * Be sure to size input fields appropriately to anticipate the longest likely value (if a maximum character length is known).
 * When a value is too long to be fully displayed in a text field, use [truncation](/design-guidelines/content/grammar-and-terminology#truncation).
 * Provide a [text area](/documentation/react/components/textarea) for input anticipated to be longer than a single line.
-
-![Text area](./img/textarea.png)
-
-* Pre-populate fields with known values when possible, e.g. a default IP address.
+* Pre-populate fields with known values when possible, like a default IP address, for example.
 * When presented in a form, the initial input field is usually the first input field required. The primary input field should receive focus when the form is presented to the user.
 
 ### Help
-**Popovers**
-Use a popover to provide context around a text or input field. Popovers are represented with a question icon and displayed when the user clicks on the icon.
+**Placeholder text**
+
+[Placeholder text](/design-guidelines/usage-and-behavior/forms#placeholder-text) is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be "DD/MM/YYYY".
 
 **Helper text**
-Helper text is text below a form field that helps a user provide the right information, like Enter a unique name.
 
-**Placeholder text**
-Placeholder text is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be e.g. DD/MM/YYYY.
+[Helper text](/design-guidelines/usage-and-behavior/forms#helper-text) is text below a form field that helps a user provide the right information, like "Enter a unique name".
+
+
+**Popovers**
+
+[Popovers](/design-guidelines/usage-and-behavior/forms#popovers) provide context around a text or input field. Popovers are represented with an unfilled question icon and displayed when the user clicks on the icon.
 
-For more information about how to provide help content, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#provide-the-right-kind-of-help).
+For more information about how to provide help content, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#user-help).
 
 ### Errors and validation
 Wherever possible, use inline errors to notify users that something has gone wrong. Inline errors help should be short and help users quickly identify and understand what they need to fix and why.
```

Comment:
Change "something has gone wrong" to "there is a problem with the user's input"?